### ISSUE TITLE
Clean the windows in the kitchen 

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#714]
Scrub the windows, ideally before the cupboard between the kitchen and nook is finished

**Comment by zarino** [comment]
We tried cleaning these windows with Mr Muscle glass cleaning spray and a cloth today, and annoyingly a film of grime remains, no matter how much you repeat the process.

I'm seeing suggestions online for making up your own window cleaner (8 parts water, 1 part vinegar, splash of washing up liquid), spraying it on, and scrubbing with a soft bristle brush.

If that doesn't work, I know _some_ cleaning creams ("bar keepers friend" etc) are safe to use on glass, and will probably do a better job of cutting through the film/scale on the surface.

**Comment by ajlennon** [comment]
The metal frames seem to be in a state of disrepair. Do we need to do something about that at some point? What would we do? Gloss paint the metal?

**Comment by magman2112** [comment]
I also have some glass cleaning products that I can bring in, to see if any can clear the grime.

For the frames, I would suggest a stiff brush, possibly a wire brush to remove any loose surface material, as a first pass, then see what we want to do next. Painting would be a significant exercise.

**Comment by drfraz** [comment]
The internal windows are very delicate at the moment. I scrubbed most of the ones in the nook & the walled off space but not the kitchen area. 
They will need some new grout as much of it is falling off. 
with wooden frames the glass is normally also secured with some nails but with metal frames I think it is just the grout, so be very careful about pushing from the north side.
The frames could be scrubbed with a wire brush, lightly sanded then sprayed with Hammerite for an easy win.

**Comment by zarino** [comment]
Update: Tried washing up liquid solution on the glass, and it wasn’t strong enough.

I'm planning to be in on Friday – I will bring some cleaning products with me, and have a go, unless I hear otherwise.

Agree that the windows need re-grouting on the south-facing sides (events room and Room 29). But I wonder whether that’s something we could delay until _after_ the craziness of the move?

**Comment by magman2112** [comment]
I believe the “grout” is just basic glazing putty. Where it is brittle, this can be removed and just replaced with normal putty. This should also be done for the few broken panes of glass in these windows.

**Comment by drfraz** [comment]
The grouting will be fairly simple, although a bit time consuming & can wait till after the move.
I could mask off the windows & spray the frames on the north side next weekend

**Comment by johnmckerrell** [comment]
If windows are loose and dangerous I’d hope we can add to snag list. 

--

**Comment by zarino** [comment]
Quick update – I’m planning to come and scrub up these windows this evening. Will report back with how it goes.

**Comment by magman2112** [comment]
@zarino I left some window cleaning spray in “Room 29” that appears to clean the windows very well, though I would suggest a good wash with soapy water first before using the cleaning fluid. I tried it on a couple of panes in this room, so see if you can spot the clean ones :-) (one of which still needs a bit more cleaning on the nook side).

The cleaner is called Nilglass and was on the wood working bench when I left it yesterday. I also left a Stanley knife blade, which can be used to scrape paint spots off the glass.

For the window frames, I would suggest a stiff brush, then a scouring pad to remove any loose surface material. On the kitchenette side, I would also suggest putting unopened bin bags down over the loose floorboards adjacent to the window, so that any spilt water doesn’t leak through to the folks downstairs.

**Comment by zarino** [comment]
@drfraz and I spent a few hours cleaning the kitchen windows last night. They’ve been soap-water washed on both sides, and maybe a dozen of them have also been cleaned both sides with glass cleaner.

Of those, about 7 have come out sparkling, the rest seem to be covered in limescale that we just can’t remove. @amcewen says he’ll bring some Viakal today, so I will try that later.

We couldn’t reach some of the windows on the events room side, because the big disassembled laser cutter is in the way, and there weren’t enough of us there last night to move it.

**Comment by drfraz** [comment]
Now the window frames, if not all the windows are clean I was planning to paint the frames on the Kitchen/ north side next weekend.
Does anyone have any objections to Black Hammerite? It doesn't require primer & the finish will help hide all the lumps & bumps.
I was also going to renew the putty at some point & would be able to remove the glass and move it around so that all the patterned glass is on the bottom 2 rows in the kitchen.
We could also Laser etch it at this point with the DoES logo. 
Any other suggestions welcome, I thought 1 pane with all the names that donated to the crowd-funder?

**Comment by zarino** [comment]
> Does anyone have any objections to Black Hammerite?

Looks like Hammerite doesn’t do dark grey, so I guess Black is the next best thing. :+1:

> remove the glass and move it around so that all the patterned glass is on the bottom 2 rows in the kitchen

I’m not so sure about this. Part of the charm is in the random distribution of the patterned panes. If we line them all up at the bottom, I think it’ll look a bit weird.

>  We could also Laser etch it at this point with the DoES logo.

Can’t remember whether I mentioned this before, but I don’t see the point of lasering our logo onto window panes. Like, literally, what’s the point?

> I thought 1 pane with all the names that donated to the crowd-funder?

This is a cute idea. Especially if we can lay it out such that the names properly _fill_ the glass, so from a distance it’ll just look like one of the other patterned panes, but up close, you will realise it’s a list of names.

All this talk of laser etching, though, feels like running before we can walk. Most of the windows are still covered in crap. As someone who’s going to be in that space 9–5 most days pretty soon, I’d rank crystal clear windows _far higher_ than novelty etchings, on my list of priorities.

What do other people think?

**Comment by ajlennon** [comment]
The cleaning and painting sounds a great idea. I do think the random distribution looks better than it would if it were rearranged.... I like the idea of the etched names somewhere... I think etched named of everybody involved in the move with a date would be a cute legacy too...

**Comment by amcewen** [comment]
The main argument I could see for arranging the patterned panes at the bottom would be that it would provide some privacy for the events room, in case people are having confidential meetings in there and don't want to be overlooked by the kitchen (assuming we eventually do something to cover the massive hole with a hatch :-)

Etching names-wise, we do have a crowdfunder reward that is aiming to do just that... would be a nice way to fulfill that reward IMHO.

**Comment by johnmckerrell** [comment]
Did we actually get around to fully cleaning the kitchen side of these? I don't know if anyone got around to moving any panes but I think it could be good to make sure the bottom 2 (?) rows were patterned for the privacy aspect. As they're all different patterns I don't think it would detract from the randomness of it.

**Comment by chris-does** [comment]
I have ordered some [Unger RubOut](https://www.ungerglobal.com/en/rubout) glass clearner from [Window Cleaning Warehouse](https://windowcleaningwarehouse.co.uk/products/unger-rubout-glass-cleaner).

I hope to be able to test this on the glass to see if it will be useful in removing the stubborn marks.

If it works well, I will aim to schedule some time to give the windows a clean.

**Comment by zarino** [comment]
@chris-does If the RubOut works, count me in for helping with the cleaning 👍

**Comment by chris-does** [comment]
Last Thursday, 28th November, I tested the [Unger RubOut](https://www.ungerglobal.com/uk/rubout) on one of the panes of glass (bottom right pane in the image below):

![Bottom right window pane cleaned with Unger RubOut by chris-does](https://i.imgur.com/aCzZT2n.jpg)

The results are not perfect, though I do think it shows a discernible improvement and one that is arguably sufficient to justify using the product on the rest of the windows:

One thing to note, I believe there is some scope for optimisation of the cleaning method used beyond my initial technique.

Some reading of window cleaning forums has yielded the following best practice:

- Wash the glass and surrounding area using warm water, ordinary dish detergent (“Fairy Liquid Original” seems popular!) using a soft clean sponge.
- Rinse the detergent off with clean water.
- Remove excess water / dry the windows with lint free paper towel or a lint free micro-fibre cloth.
- Apply a small amount of [Unger RubOut](https://www.ungerglobal.com/uk/rubout) to the rough/scouring side of a, slightly damp, combined scourer/sponge such as 
![GitHawk Upload by chris-does](https://i.imgur.com/C5eiTd7.jpg)
- Use overlapping circular motions with the rough/scouring side of the scourer/sponge to “RubOut” the stains/dirt on the glass. 
- Once the product has been worked evenly over the entire glass surface several times, it should start to dry somewhat cloudy.
- Remove the cloudy remnants of the product with a dry lint free paper towel or lint free micro fibre cloth - again using overlapping circular motions.
- Repeat the above process for particularly stubborn dirt or stains.
- Once satisfied with the “RubOut” results, finish the glass by using Windolene and a dry lint free paper towel or lint free micro fibre cloth according to the manufacturer’s instructions.

Given the potential for water drips/spillage etc. if doing the remainder of the glass, especially those panes that are near people’s desks and or electrical items, I wonder if we might need to schedule a specific day to clean - at least the panes with the greatest risk. I certainly don’t want to spill soapy water anywhere near @johnmckerrell’s (or anyone else’s) desk! :-)

The remainder of the panes could be done on a more ad-hoc basis :-)

Thoughts welcome . . .
 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jackie1050** [comment]
We're aiming to arrange a Hack the Space day in between Xmas and the New Year - there might be fewer people co-working then so should be a good time for this.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by johnmckerrell** [comment]
I actually thought the windows on that side had been done and it was just the ones on the kitchen side that needed doing. 

--

**Comment by amcewen** [comment]
Seven years in, I think we re-open this or create a new issue if/when we decide to do more window cleaning.
